Packs/CommonScripts/Scripts/IsIPPrivate/README.md

```diff
@@ -0,0 +1,7 @@
+The script takes one or more IP addresses and checks whether they're in the private IP ranges defined in the *PrivateIPsListName* argument. By default, the *PrivateIPsListName* argument will use the Cortex XSOAR list called "PrivateIPs".
```

Comment:
why not `demisto-sdk generate-docs`?

Reply:
Because I'm getting an error with it 
![image](https://user-images.githubusercontent.com/43602124/219953164-a767b015-9f9e-41b9-85a9-09ec7592cc69.png)